tests/test_value_propagation.py

```diff
@@ -245,3 +245,16 @@ def test_strings():
 
     x, y = op.const(["foo"]), op.const(["bar"])
     np.testing.assert_equal(op.string_concat(x, y)._value.value, np.array(["foobar"]))  # type: ignore
+
+
+@pytest.mark.parametrize("dtype", [np.int32, np.float32])
+def test_tensor_raw_bytes(dtype):
+    # - Test on integers and floats to check for possible issues with
+    # endianess.
```

Comment:
I don't see how this checks any "endianess" issues? You are turning these into ONNX tensors and recovering them back on the same platform. It also doesn't look like something that belongs as a Spox unit test but rather the upstream onnx library, if it isn't already tested.

Reply:
The data still goes through onnxruntime. Nonetheless, you are right that this does not fall into Spox's responsibility, and things appear to work as expected anyhow. I removed the test.